public/css/list/incident-list.less

```diff
@@ -0,0 +1,6 @@
+.incident-list .list-item {
```

Comment:
This doesn't seem to be specific to the incident list. The event list could also need it, so please move this to common.less